docs/api.md

```diff
@@ -181,3 +181,18 @@ Example (generated using the body and public key from the previous examples):
     }
 }
 ```
+
+## Authenticated Media
+
+When accessing media from a Synapse homeserver with authenticated media enabled, an additional
+HTTP header needs to be passed along with any request to the Matrix Content Scanner.
+
+This header follows the Matrix specification `Authentication Bearer scheme` as outlined [here](https://spec.matrix.org/v1.12/client-server-api/#using-access-tokens).
+The `Authorization` header method must be used, sending the access token as a query string
```

Comment:
Folks might be confused about *what* access token they should use. You may want to say that it should be the Matrix access token of the client's user.